### PR TITLE
Add jumas layout

### LINKED ISSUE #63: Add Jumua layout
The idea is that Jumas are excluded from the prayer layout. We discussed putting them in their own layout. 
When we implement #9, we can default to this layout on Fridays (starting after latest Isha on Thursday and ending after latest Juma on Friday).

### PR DESCRIPTION
**Pull request by fayez-nazzal**
Add new layout for jumas:
- Includes the masjid name followed by a list of Jumas
- The masjid name links to the main layout

## Screenshot

image

### CODE REVIEW

**Comment by fayez-nazzal** [comment]
Need to check a real most recent notified.json @hammady

**Comment by hammady** [comment]
@fayez-nazzal Updated the linked issue.

File: generator/svelte/src/lib/types.ts [hunk 1]
```diff
@@ -26,6 +26,6 @@ export interface IMasjid {
 }
 
 export type TMasjidsJSON = {
-	max_jumaas: number;
+	max_jumas: number;
```
Comment (hammady) [inline generator/svelte/src/lib/types.ts]: This is now deprecated.

Comment (fayez-nazzal) [reply generator/svelte/src/lib/types.ts]: Removed.

**Review by hammady — APPROVED** [review]

**Review by hammady — CHANGES_REQUESTED** [review]
Please add the last updated text somewhere below the masjid name.

**Comment by fayez-nazzal** [comment]
> Please add the last updated text somewhere below the masjid name.

Added.

**Review by hammady — APPROVED** [review]

**Pull request merged**